interfaces/seccomp/template.go

```diff
@@ -161,9 +161,9 @@ inotify_init
 inotify_init1
 inotify_rm_watch
 
-# TIOCSTI allows for faking input (man tty_ioctl)
-# TODO: this should be scaled back even more
-ioctl - !TIOCSTI
+# FIXME: replace this with the filter of TIOCSTI once snap-confine can read this syntax
+# See LP:#1662489 for context.
+ioctl
```

Comment:
This seems like the wrong approach. We will never be able to adjust seccomp policy that requires changes to snap-confine if we wait for 'snap-confine can read this syntax' since a package might be stuck for a long time. What is wrong with the exec'ing snap-confine from core approach?